docs/dev/developer-guides/transactions/transactions.md

```diff
@@ -1,61 +1,71 @@
+## Transaction data on Ethereum
 
-## How transactions work
+From [Ethereum.org](https://ethereum.org/en/developers/docs/transactions/):
 
-When a user initiates a transaction on Ethereum, some specific data is created:
+> A submitted transaction includes the following information:
+>  - `from`: the address of the sender, that will be signing the transaction. This will be an externally-owned account as contract accounts cannot send transactions.
+>  - `recipient`: the receiving address (if an externally-owned account, the transaction will transfer value. If a contract account, the transaction will execute the contract code).
+>  - `signature`: the identifier of the sender. This is generated when the sender's private key signs the transaction and confirms the sender has authorized this transaction.
+>  - `nonce`: a sequentially incrementing counter which indicates the transaction number from the account.
+>  - `value`: amount of ETH to transfer from sender to recipient (denominated in WEI, where 1ETH equals 1e+18wei).
+>  - `data`: optional field to include arbitrary data.
+>  - `gasLimit`: the maximum amount of gas units that can be consumed by the transaction. The EVM(opens in a new tab)↗ specifies the units of gas required by each computational step.
+>  - `maxPriorityFeePerGas`: the maximum price of the consumed gas to be included as a tip to the validator.
+>  - `maxFeePerGas`: the maximum fee per unit of gas willing to be paid for the transaction (inclusive of baseFeePerGas and maxPriorityFeePerGas).
 
-- **Receiver**: The recipient is the account's address to receive the transaction. The receiver can be a contract account or an externally owned account. Each transaction is aimed toward a specific recipient.
-- **Nonce**: This field displays the most recent transaction based on the account's counter, which maintains track of how many transactions it does. The network uses the transaction nonce to ensure that transactions are completed in the correct sequence.
-- **Gas price**: Most transactions necessitate the payment of a fee from the transaction's author. This cost is computed per unit of gas. The unit is Wei, a smaller ether unit.
-- **Gas limit**: The transaction author specifies the number of gas units used for the transaction. This is the total amount of gas that could be consumed.
-- **Value**: The quantity of Wei or Ether that the sender account wishes to transmit to the recipient is represented by the value.
-- **Data**: If the transaction receiver is a smart contract, the data contains information for the contract's functions to be executed. This comprises data with varying lengths.
-- **Signature**: A signature indicates who sent the communication. The signature is created when an externally owned account confirms and signs the transaction with its private key.
+## Nature of transactions
```

Comment:
Please revise this section. It is not well written, and I don't understand the flow using bullet points

Reply:
yep had a go